src/database/schema/ChannelListSchema.ts

```diff
@@ -118,8 +122,8 @@ class ChannelList implements BaseDbSchema {
     myAnonymousId: string
   ): Promise<ChannelList> => {
     const channel = await this.getById(db, channelId);
-    const members = await ChannelListMemberSchema.getAll(db, channelId, myId, myAnonymousId);
-    channel.members = members;
+    const memberUsers = await UserSchema.getAllByChannelId(db, channelId, myId, myAnonymousId);
+    channel.memberUsers = memberUsers;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `getById` method has been modified to fetch `memberUsers` using the `UserSchema.getAllByChannelId()` method instead of `ChannelListMemberSchema.getAll()`. This change aligns with the introduction of the `memberUsers` property.
```diff
-    const members = await ChannelListMemberSchema.getAll(db, channelId, myId, myAnonymousId);
-    channel.members = members;
+    const memberUsers = await UserSchema.getAllByChannelId(db, channelId, myId, myAnonymousId);
+    channel.memberUsers = memberUsers;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->